doc/method/sources.md

```diff
@@ -1,77 +1,169 @@
+We build the taxonomy from nine sources.
+OTT assembly is dependent on the input order of the sources - higher ranked inputs take priority over lower ranked inputs.
+[see table]
 
-*NCBI*  
-The first objective of the Open Tree taxonomy is to
-align OTUs across phylogenetic studies.  This need is largely met by
-using the NCBI taxonomy, since (1) most phylogenies are molecular, (2)
-publishers require molecular sequences used in studies to be deposited
-in Genbank, (3) every Genbank deposit is annotated with a taxon from
-NCBI taxonomy.  NCBI also tends to be more phylogenetically-informed than other taxonomies (see Results, below), which makes it a good candidate as a default backbone. The NCBI taxonomy therefore forms the nucleus of OTT.
+[The following should be a table:
 
-The disadvantage of the NCBI taxonomy is that it has only [NNN] species
-records with standard binomial names. This represents taxa that have sequence information. NCBI therefore does not alone provide sufficient coverage for OpenTree.
+ * the ARB-SILVA taxonomy
+ * Hibbett 2007
+ * Index Fungorum 
+ * Schaferhoff 2010
+ * WoRMS
+ * the NCBI taxonomy
+ * the GBIF backbone taxonomy
+ * IRMNG
+ * additional taxa provided by Open Tree study curators
 
-The particular version of NCBI taxonomy used in the OTT assembly described below (OTT 2.10) was downloaded on
-[date], but we have retrieved a fresh version with every OTT build.
+References for all sources could be a column in the table.
 
-*GBIF*  
-The GBIF taxonomy provides much greater coverage, in terms of number of binomials, than NCBI. The GBIF backbone draws from a number of sources (including
-IRMNG and Index Fungorum), has ongoing institutional support, and no
-special terms of use, making it a good choice for the goal of coverage. It provides much of the content available in Catalog of Life, but with terms of use that are suitable  for OpenTree.
+The root clade could be a column in the table.
 
-The GBIF version used in OTT 2.10 was released in July 2003.  A successor was released on [date], as this report was being prepared, and is scheduled for incorporation into OTT.
+The ranking can be a column in the table (although if the table is in
+rank order, it would just be sequential numbers).
 
-*SILVA*  
-The classification of prokaryotes and unicellular Eukaryotes in NCBI and GBIF is not consistent with current phylogenetic thinking. We therefore imported the SILVA taxonomy, which is built algorithmically based on clustering of molecular
-sequences.  We incorporated SILVA version 115 into OTT 2.10.  We did
-not include SILVA's plant, animal, or fungi branches in OTT
-[references]. SILVA has higher priority than NCBI or GBIF in order to capture the deep relationships in the tree.
+The following information might also go in the table: total number of
+taxa, number of exclusive taxa.
```

Comment:
Some good suggestions here for things to include. Too much for one table, so one way to prune is to include only basic stats here, not results (i.e. number of taxa = yes; exclusive taxa = no).